lib/repositories/workspaces_repository.dart

```diff
@@ -60,7 +55,7 @@ class WorkspacesRepository {
 
     _storage.multiInsert(table: Table.workspace, data: workspaces);
```

Comment:
What the point in reinserting the workspaces, you just received from local database? It seems you forgot to fetch it from remote server first...